mtz-update-password.html

```diff
@@ -256,6 +319,19 @@
         retypePlaceholder: String,
         /* The icon to use when a status message is valid */
         validStatusIcon: String,
+        /* Strength score and corresponding text to display */
+        strength: {
+          type: Object,
+          value() {
+            return {
+              0: 'Too short ☹',
```

Comment:
Remove the Emojis, they aren't going to work consistently across browsers / fontsets, and it's easy enough for the user to add in their own that will work with their chosen fontset

Reply:
Ok

Reply:
@stramel I’ll see if there is an alternative or we could build a simple validator.